src/main/java/de/adesso/service/MarkdownTransformer.java

```diff
@@ -62,17 +62,24 @@ private String getLastPartOfPath(String path) {
     }
 
     private void copyFile(Path sourcePath, String datePart) {
-        String destinationPath = datePart + "/" + datePart + "-" + sourcePath.getFileName().toString();
+        Path destinationPath = new File(configService.getFIRSTSPIRIT_XML_PATH() + "/" + datePart + "/" + datePart + "-" + sourcePath.getFileName().toString()).toPath();
         try {
             LOGGER.debug("Copy file from " + sourcePath + " to " + destinationPath);
-            Files.copy(sourcePath, new File(configService.getFIRSTSPIRIT_XML_PATH() + "/" + destinationPath).toPath(), StandardCopyOption.REPLACE_EXISTING);
+            if (pathDoesNotExist(destinationPath)) {
+                Files.createDirectories(destinationPath);
+            }
+            Files.copy(sourcePath, destinationPath, StandardCopyOption.REPLACE_EXISTING);
         } catch (IOException e) {
             LOGGER.error("An error occurred while copying generated XML files to destination.", e);
             LOGGER.error("Exiting jekyll2cms.");
             System.exit(36);
```

Comment:
- In die Exitmethode auslagern
- und bitte angeben, was der Fehler ist. Dazu am besten wie man ihn behebt :) In dem Fall "<Specified> path does not exist. Provide <lösung-des-problems>" Bitte auch kurz angeben was <Specified> hier ist. Elternpfad oder so? Hab ich noch nicht ganz verstanden.

Reply:
Eine Exit-Methode an einer statischen Klasse existiert in diesem Projekt noch nicht, kann aber per neuem Feature eingeführt werden.

Die IOException kann aus mehreren Gründen geworfen werden. Dazu gehören: 
* Source Path ist kein korrekter Pfad
* Source Path existiert nicht
* Destination Path ist kein korrekter Pfad

Auf alles innerhalb der Logger-Meldung einzugehen halte ich für zu viel. Dadurch, dass die Error-Message mit ausgegeben wird, sollten diese Informationen gegeben sein. Bspw:
```
An error ocurred while Copying generated XML files to destination.
NoSuchFileException: <sourcePath>
```

Was meinst du mit hier?

Reply:
Entschuldige ich habe das Repository mit dem blogpost-checker vertauscht!
Ich mach ein Issue auf.